Zend/zend_compile.c

```diff
@@ -339,6 +339,7 @@ void zend_oparray_context_begin(zend_oparray_context *prev_context, zend_op_arra
 	CG(context).try_catch_offset = -1;
 	CG(context).current_brk_cont = -1;
 	CG(context).last_brk_cont = 0;
+	CG(has_assigned_to_http_response_header) = false;
```

Comment:
Hm, doesn't this need to be on the op array context instead? Otherwise we only warn once per compilation instead of once per file?

Reply:
Yeah okay I see I actually think I wanted to say "once per op_array", but if that's the intention that's fine I guess?

Reply:
But it seems to be once per op_array :thinking:

Reply:
I'll check this again tomorrow when I'm more awake